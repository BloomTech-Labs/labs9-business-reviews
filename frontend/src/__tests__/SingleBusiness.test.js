import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import SingleBusinessComponent from '../components/SingleBusiness';

describe('SingleBusinessComponent />', () => {
  it('renders', () => {
    shallow(<SingleBusinessComponent />);
  });
  it('matches snapshot', () => {
    const wrapper = shallow(<SingleBusinessComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
