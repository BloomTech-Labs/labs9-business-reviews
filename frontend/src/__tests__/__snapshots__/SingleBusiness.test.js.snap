// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SingleBusinessComponent /> matches snapshot 1`] = `
<styled.div
  className="single-business"
>
  <div
    className="modal"
  >
    <h1>
      Business Name
    </h1>
    <img
      alt="business"
      src=""
    />
    <div
      className="reviews-container"
    >
      <h4>
        review title
      </h4>
      <p>
        review body
      </p>
      <h4>
        review title
      </h4>
      <p>
        review body
      </p>
      <h4>
        review title
      </h4>
      <p>
        review body
      </p>
      <h4>
        review title
      </h4>
      <p>
        review body
      </p>
      <h4>
        review title
      </h4>
      <p>
        review body
      </p>
    </div>
    <button>
      Close
    </button>
  </div>
</styled.div>
`;
